client/src/pages/course/student/cross-check-review.tsx

```diff
@@ -180,11 +154,27 @@ function Page(props: CoursePageProps) {
       return;
     }
 
-    if (checkPoints().length !== countStar.length) {
-      return notFilledCriteriaWarning();
+    const criteriaToSubmit = criteriaData.map(item => {
+      if (item.type !== TaskType.Title && !item.point) {
+        item.point = 0;
+      }
+      return item;
+    });
+
+    setCriteriaData(criteriaToSubmit);
+
+    const isCriteriaPointsAndCommentsVerified = criteriaToSubmit
+      .filter(criteria => criteria.type.toLowerCase() === TaskType.Subtask)
+      .every(item => {
+        return item.point === item.max ? true : item.textComment && item.textComment.length > 10;
```

Comment:
I've seen above check that comment should be **at least** 10 symbols length, but here you check that comment should be **longer** that 10 symbols. Is this correct?

Reply:
Thanks, you are so attentive :)
I completely missed this moment